refarch-integrations/refarch-dms-integration/refarch-dms-integration-core/src/test/java/de/muenchen/refarch/integration/dms/adapter/out/fabasoft/DMSErrorHandlerTest.java

```diff
@@ -50,9 +50,10 @@ void handleDmsException() {
     }
 
     @Test
+    @SuppressWarnings("PMD.UnitTestShouldIncludeAssert")
```

Comment:
I agree with PMD this is not a valid unit test here. Even though you are testing the positive case we should still include assertions to validate the positive outcome.

Reply:
Refactored